.github/workflows/ubuntu-devel.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   main:
-    name: "Test: Python 3.9"
+    name: "Test: Python 3.10"
```

Comment:
According to https://packages.ubuntu.com/search?keywords=python3, the Ubuntu devel version only have 3.9 yet.

Reply:
I've reverted the changes to this file :+1: